src/meshcat/visualizer.py

```diff
@@ -176,6 +176,9 @@ def set_object(self, geometry, material=None):
     def set_transform(self, matrix=np.eye(4)):
         return self.window.send(SetTransform(matrix, self.path))
 
+    def set_property(self, path, key, value):
```

Comment:
(copied from email)

This should use `self.path` rather than taking a `path` string argument. To set the property of an object in the scene, you'd do:

```py
vis["foo"]["bar"].set_property(key, value)
```

just like `set_transform` and `set_object`.